dashboard-widgets.el

```diff
@@ -664,74 +669,98 @@ WIDGET-PARAMS are passed to the \"widget-create\" function."
 ;;
 ;; Org Agenda
 ;;
-(defun dashboard-timestamp-to-gregorian-date (timestamp)
-  "Convert TIMESTAMP to a gregorian date.
-
-The result can be used with functions like
-`calendar-date-compare'."
-  (let ((decoded-timestamp (decode-time timestamp)))
-    (list (nth 4 decoded-timestamp)
-          (nth 3 decoded-timestamp)
-          (nth 5 decoded-timestamp))))
-
-(defun dashboard-date-due-p (timestamp &optional due-date)
-  "Check if TIMESTAMP is today or in the past.
-
-If DUE-DATE is nil, compare TIMESTAMP to today; otherwise,
-compare to the date in DUE-DATE.
-
-The time part of both TIMESTAMP and DUE-DATE is ignored, only the
-date part is considered."
-  (unless due-date
-    (setq due-date (current-time)))
-  (setq due-date (time-add due-date 86400))
-  (let* ((gregorian-date (dashboard-timestamp-to-gregorian-date timestamp))
-         (gregorian-due-date (dashboard-timestamp-to-gregorian-date due-date)))
-    (calendar-date-compare (list gregorian-date)
-                           (list gregorian-due-date))))
+(defcustom dashboard-week-agenda
+  t
```

Comment:
I think for this kind of little code, we can just put `t` at the end of the variable name. Thanks!

Also for,

* `dashboard-week-agenda`
* `dashboard-agenda-time-string-format`
* `dashboard-match-agenda-entry`